src/plugins/midi-piano/midi-piano-display.js

```diff
@@ -0,0 +1,292 @@
+import axios from 'axios';
+import * as Tone from 'tone';
```

Comment:
I got rid of the key range selector for now. The strange behaviour came from the react-piano mouse event handlers. It actually selected the notes you clicked on, but the visualization was not correct.
The crash was due to a bug I missed since I didn't actually save right after using the key selector.